v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -259,6 +259,26 @@ func init() {
 		"contains": makeDslFunction(2, func(args ...interface{}) (interface{}, error) {
 			return strings.Contains(types.ToString(args[0]), types.ToString(args[1])), nil
 		}),
+		"contains_all":makeDslWithOptionalArgsFunction(
+			"(body interface{}, substrrs ...string) bool",
+			func(arguments ...interface{}) (interface{}, error) {
+				for _, value := range arguments[1:] {
+					if !strings.Contains(types.ToString(arguments[0]), types.ToString(value)) {
```

Comment:
`types.ToString(arguments[0])` can be stored into a variable before the loop to avoid conversion at each iteration

Reply:
Done